app/controller/sdl/RCModulesController.js

```diff
@@ -44,7 +44,7 @@ SDL.RCModulesController = Em.Object.create({
       LIGHT: 'lightModels',
       HMI_SETTINGS: 'hmiSettingsModels'
     },
-
+    
```

Comment:
@YarikMamykin try to remove these spaces as well

Reply:
@AKalinich-Luxoft removed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/3c8362dcc12e893e440a5fe537d926124c10eb3d